tests/integration/conftest.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from pathlib import Path
+
+import pytest
+import yaml
+
+
+@pytest.fixture
+def metadata():
+    metadata = Path("./metadata.yaml")
+    data = yaml.safe_load(metadata.read_text())
+    return data
+
+
+@pytest.fixture
+def model(ops_test):
+    return ops_test.model
```

Comment:
In the [Contributing Guide](https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#handling-typing-issues-with-python-libjuju) there is a "assert ops_test.model" before returning it.

Reply:
Adding integration tests is a part of the current story. Will be changing it.